gobblin-runtime/src/test/java/gobblin/scheduler/JobConfigFileMonitorTest.java

```diff
@@ -63,9 +67,14 @@ public Integer apply(Void input) {
   @BeforeClass
   public void setUp()
       throws Exception {
+    final Logger log = LoggerFactory.getLogger("setUp");
+    this.jobConfigDir = Files.createTempDirectory("gobblin-test_job-conf").toString();
```

Comment:
can you add something about the class name to the temp dir name like `gobblin-test-job-config-file-monitor-test`